homeassistant/components/light/decora_wifi.py

```diff
@@ -0,0 +1,245 @@
+"""
+Interfaces with the myLeviton API for Decora Smart WiFi products.
+
+See:
+http://www.leviton.com/en/products/lighting-controls/decora-smart-with-wifi
+
+Uses Leviton's cloud services API for cloud-to-cloud integration.
+
+"""
+
+import json
+import logging
+import requests
```

Comment:
Sorry, don't understand this comment - it's already above voluptuous. Can you clarify please?